examples/topology.example.yaml

```diff
@@ -211,6 +211,54 @@ tiflash_servers:
 #   - host: 10.0.1.21
 #   - host: 10.0.1.22
 
+# NOTE: Only 1 master node is supported for now
+tispark_masters:
+  - host: 10.0.1.21
+    #spark_config:
+    #  spark.driver.memory: "2g"
+    #  spark.eventLog.enabled: "False"
+    #  spark.tispark.grpc.framesize: 268435456
+    #  spark.tispark.grpc.timeout_in_sec: 100
+    #  spark.tispark.meta.reload_period_in_sec: 60
+    #  spark.tispark.request.command.priority: "Low"
+    #  spark.tispark.table.scan_concurrency: 256
+    #spark_env:
+    #  SPARK_EXECUTOR_CORES: 5
+    #  SPARK_EXECUTOR_MEMORY: "10g"
+    #  SPARK_WORKER_CORES: 5
+    #  SPARK_WORKER_MEMORY: "10g"
+
+# NOTE: multiple worker nodes on the same host is not supported
+tispark_workers:
+  - host: 10.0.1.22
+    #spark_config:
+    #  spark.driver.memory: "2g"
+    #  spark.eventLog.enabled: "False"
+    #  spark.tispark.grpc.framesize: 268435456
+    #  spark.tispark.grpc.timeout_in_sec: 100
+    #  spark.tispark.meta.reload_period_in_sec: 60
+    #  spark.tispark.request.command.priority: "Low"
+    #  spark.tispark.table.scan_concurrency: 256
+    #spark_env:
```

Comment:
Prefer to `env`.

Reply:
I think spark_env should not be set for workers.